src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related
+ * CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';')
+    .map(element => element.trim().split(':'))
+    .filter(arr => arr.length > 1);
+
+  const callback = (acc, element) =>
+    ({
+      ...acc, [element[0].trim()]: element[1].trim(),
+    });
```

Comment:
If it is a callback function, will be better to move it outside of your convertToObject function, to avoid creating it on each call.